From 8e4960e8b9d11d9be96e83b02bd3b08178ddce1b Mon Sep 17 00:00:00 2001
From: Sergey Bobrenok <bobrofon@gmail.com>
Date: Wed, 28 Feb 2024 20:42:19 +0300
Subject: [PATCH basu] memfd-util.c: Use F_ADD_SEALS definition from missing.h

uClibc and glibc (until version 2.27, 2018) implementations don't
support F_ADD_SEALS and other F_SEAL_xxx flags.
'missing.h' was removed from 'memfd-util.c' in 37dbb2fcb ("Remove
useless includes").

Upstream: https://lists.sr.ht/~emersion/public-inbox/patches/49899

Signed-off-by: Sergey Bobrenok <bobrofon@gmail.com>
---
 src/basic/memfd-util.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/basic/memfd-util.c b/src/basic/memfd-util.c
index 89893ba..7e535c6 100644
--- a/src/basic/memfd-util.c
+++ b/src/basic/memfd-util.c
@@ -7,6 +7,7 @@
 
 #include "alloc-util.h"
 #include "memfd-util.h"
+#include "missing.h"
 
 int memfd_set_sealed(int fd) {
 #if defined(__FreeBSD__) && __FreeBSD__ < 13

base-commit: 684a41d68cfbb05e38aacb60a8548e21ddfbecdb
-- 
2.44.0

